assets/clients/clients.yaml

```
@@ -394,6 +394,17 @@ clients:
       - icon: github
         url: https://github.com/dragonflylee/switchfin/releases/latest
 
+  - name: "tsukimi"
+    targets: [ Linux ]
```

I think it also has Windows binaries:
https://github.com/tsukinaha/tsukimi/releases

```suggestion
    targets: [ Linux, Windows ]
```

---

huh, you are correct! i only saw its bio "A simple third-party Emby client for Linux" so yup :sweat_smile: